nuclear/module/NUClearModule.cmake

```diff
@@ -191,6 +191,16 @@ function(NUCLEAR_MODULE)
   # Put it in an IDE group for the module's directory
   set_target_properties(${module_target_name} PROPERTIES FOLDER ${module_path})
 
+  # ####################################################################################################################
+  # Warnings #
+  # ####################################################################################################################
+
+  # * We need to tell cmake where the function for the compiler warnings is (src/cmake)
+  list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/nuclear/cmake")
```

Comment:
Move this line [here](https://github.com/NUbots/NUbots/blob/10337a29ed89d30b44563b83bee487141de0c9a3/nuclear/CMakeLists.txt#L2)